app/models/host_aggregate.rb

```diff
@@ -0,0 +1,43 @@
+class HostAggregate < ApplicationRecord
+  include NewWithTypeStiMixin
+  include Metric::CiMixin
+  include EventMixin
+  include ProviderObjectMixin
+  include VirtualTotalMixin
+
+  acts_as_miq_taggable
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+
+  # FIXME: I don't think this is needed here
```

Comment:
I left the FIXMEs for the initial review to see if  there were any comments that specific ones were needed. These comments will all be gone when I remove WIP from this. I know that I need to know VMs for rollups, etc, but should this be an explicit relation (that must be maintained when VMs are created/destroyed and hosts are added/removed), or can the UI and rollups code handle this as just a method that iterates over the hosts and calls vms on each?